Documenting support of custom certificates (#9094)
DataDog/documentation: content/en/synthetics/private_locations/configuration.md

| `concurrency` | Number | `10` | Maximum number of tests executed in parallel. |
| `maxTimeout` | Number | `60000` | Maximum test execution duration for API tests (in milliseconds). |

## Private Certificates

You can upload custom certificates to your private locations to have your API and Browser tests perform SSL handshake using your own `.pem` files. To do so:

> * Mount your certificates (ie. all the relevant `.pem` files) to `/etc/datadog/certs` when spinning up your private location containers. This can be done the same way your private location configuration file is mounted. 

Author: ```suggestion
* Mount your certificates (all the relevant `.pem` files) to `/etc/datadog/certs` when spinning up your private location containers. This can be done the same way your private location configuration file is mounted. 
```

> You can upload custom certificates to your private locations to have your API and Browser tests perform SSL handshake using your own `.pem` files. To do so:
> 
> * Mount your certificates (all the relevant `.pem` files) to `/etc/datadog/certs` when spinning up your private location containers. This can be done the same way your private location configuration file is mounted. 
> * These certificates are then considered trusted CA and used as such at test runtime;

Reviewer: ```suggestion
You can upload custom certificates to your private locations to have your API and Browser tests perform SSL handshake using your own `.pem` files. When spinning up your private location containers, mount the relevant certificate `.pem` files to `/etc/datadog/certs`, the same way your private location configuration file is mounted. These certificates are then considered trusted CA and used as such at test runtime.
```

> * These certificates will then be considered trusted CA and used as such at test runtime;

Author: ```suggestion
* These certificates are then considered trusted CA and used as such at test runtime;
```

> **Note**: This feature is supported for versions >1.5.3 of the image.

Reviewer: "the image" seems kind of vague to me. What's the name of the thing they are considering the version number for here?

Author: Right, what about:

```suggestion
**Note**: This feature is supported for versions >1.5.3 of the private location Docker image.
```
?

Reviewer: Yes!
